src/Schema.php

```diff
@@ -46,6 +63,24 @@ public static function uninstall() {
     Admin::removeAccessCapability();
   }
 
+  /**
+   * Ensures dynamic response headers for all requests in htaccess.
+   */
+  public static function ensureResponseHeadersHtaccess() {
+    $headers = self::HTTP_RESPONSE_HEADERS;
+    if(defined('CORE_STANDARDS_HTTP_HEADERS') && is_array(CORE_STANDARDS_HTTP_HEADERS)) {
+      $headers = array_merge($headers, CORE_STANDARDS_HTTP_HEADERS);
+    }
+    $template = '# BEGIN core-standards:headers' . PHP_EOL;
+    $template .= '<IfModule mod_headers.c>' . PHP_EOL;
+    foreach ($headers as $header => $value) {
+      $template .= "\tHeader set $header $value" . PHP_EOL;
+    }
+    $template .= '</IfModule>' . PHP_EOL;
+    $template .= '# END core-standards:headers' . PHP_EOL;
```

Comment:
All files are using Unix line endings, also on Windows. Therefore, we do not use PHP_EOL, but simply "\n". Using a literal line break (by pressing the enter key) is frequently also a good option.

Also, we use two spaces instead of tabs also in .htaccess.
```suggestion
    $template = "# BEGIN core-standards:headers\n";
    $template .= "<IfModule mod_headers.c>\n";
    foreach ($headers as $header => $value) {
      $template .= "  Header set $header $value\n";
    }
    $template .= "</IfModule>\n";
    $template .= "# END core-standards:headers\n";
```